src/vs/workbench/contrib/terminal/browser/terminalEditorService.ts

```diff
@@ -167,7 +167,7 @@ export class TerminalEditorService extends Disposable implements ITerminalEditor
 		if (resource) {
 			await this._editorService.openEditor({
 				resource,
-				description: instance.description || instance.shellLaunchConfig.description,
+				description: instance.description || instance.shellLaunchConfig.type,
```

Comment:
This needs to be the localized one, but I think that's already covered by `instance.description` now?

Reply:
yes it's now covered